src/core/logging/logger.ts

```diff
@@ -10,22 +11,35 @@ export class Logger {
 	private botConfig: BotConfig;
 	public discordLogger?: DiscordLogger;
 	public slackLogger?: SlackLogger;
+	private telegramLogger?: TelegramLogger;
 
 	// Codes that are not sent to external sources (discord, slack)
-	private externalExemptCodes: Array<number> = [4, 5, 6, 8];
+	private externalExemptCodes: Array<number> = [];
```

Comment:
Why have this line if it's empty by default and there's no env var to change it?